### PR TITLE
SW-5626 Add application permissions

### PR DESCRIPTION
**Pull request by sgrimm**

### CODE REVIEW

**Comment by sgrimm** [comment]
* **#2166** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2166?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2165** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2165?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2164** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2164?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2163** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2163?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2162** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2162?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @sgrimm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt [hunk 1]
```diff
@@ -530,6 +539,9 @@ data class IndividualUser(
   override fun canUpdateAccession(accessionId: AccessionId) =
       isManagerOrHigher(parentStore.getFacilityId(accessionId))
 
+  override fun canUpdateApplication(applicationId: ApplicationId) =
```
Comment (tommylau523) [inline src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt]: Should TF expert be able to edit applications? I am guessing they can use the review privilege to update comments and feedback.

**Review by tommylau523 — APPROVED** [review]

File: src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt (hunk 1 above)
Comment (sgrimm) [reply src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt]: Yes, the review privilege covers admin-only edits. But your comment does highlight that this name is confusing. Will figure out a clearer name.

Comment (sgrimm) [reply src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt]: I split this into two permissions, one for updating the boundary (which TF Expert should be able to do) and one for updating the submission status, which covers the "submit" and "restart" operations and which only the org admin can do.

Comment (tommylau523) [reply src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt]: Makes sense to me! Thanks!

**Review by tommylau523 — APPROVED** [review]

**Comment by sgrimm** [comment]
### Merge activity

* **Jul 17, 2:41 PM EDT**: @sgrimm started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/2162).
* **Jul 17, 2:41 PM EDT**: @sgrimm merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/2162).

**Pull request merged**